tests/test_resample.py

```diff
@@ -0,0 +1,35 @@
+import numpy as np
+
+from examples.seismic import TimeAxis, RickerSource, demo_model
+
+
+def test_resample():
```

Comment:
in the long term, should we expect more tests in this ``test_resample.py`` file?

Reply:
yes